spec/spec_helper.rb

```diff
@@ -32,6 +32,10 @@ def image_path(path, options = {})
 # include the manageiq-gems-pending matchers
 Dir[ManageIQ::Gems::Pending.root.join("spec/support/custom_matchers/*.rb")].each { |f| require f }
 
+# For embedded ansible to use external ansible tower's specs
+external_tower_gem_dir = Gem::Specification.find_by_name("manageiq-providers-ansible_tower").gem_dir
+Dir[File.join(external_tower_gem_dir, 'spec/support/**/*.rb')].each { |f| require f }
```

Comment:
@Fryguy @blomquisg how about moving the embedded ansible specs and code to the tower provider gem? I mean for the foreseeable future embedded will be very much like a tower provider